Source/Scene/Cesium3DTile.js

```diff
@@ -136,7 +150,10 @@ define([
             var type = url.substring(url.lastIndexOf('.') + 1);
             var contentFactory = Cesium3DTileContentProviderFactory[type];
 
-            if (defined(contentFactory)) {
+            if (type === 'json') {
```

Comment:
I was referring to the actual web response content-type header.  However, looking at this some more I think it's okay as long as the spec states that urls must end in the correct extension.  Terrain does something similar and requires all tiles end in `.terrain`.